triage/triage-team.rst

```diff
@@ -46,9 +42,9 @@ following:
 If a triager has any doubt about whether to close a PR, they should consult a core
 developer before taking any action.
 
-Triagers can also make use of the ``invalid`` and ``stale`` labels to suggest that a
+Triagers can also make use of the :gh-label:`pending` and :gh-label:`stale` labels to suggest that a
 PR may be suitable for closure. For more information, see the
-:ref:`GitHub PR labels <github-pr-labels>` section.
+:ref:`other` section.
```

Comment:
```suggestion
entries in the :ref:`labels-other` section.
```

* Use the current, unambiguous target label (and keeping the fragment-id-only ones separate, so they can be replaced later.
* Be clear that we are referring to the corresponding entries in this section, not the top-level section itself